src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/containerized-images.mdx

```diff
@@ -0,0 +1,86 @@
+
+Here's a guide to adding our pre-built layer to your code. 
+
+<Steps>
+<Step>
+### Add New Relic to your Dockerfile:
+New Relic offers pre-built images that include our layer's different runtimes: Java, Node.js, Python, and extension-only.
+
+1. Select the layer compatible with your specific Lambda function's runtime and region:
+ * [Java](https://gallery.ecr.aws/newrelic-lambda-layers-for-docker/newrelic-lambda-layers-java)
+ * [Node.js](https://gallery.ecr.aws/newrelic-lambda-layers-for-docker/newrelic-lambda-layers-nodejs) 
+ * [Python](https://gallery.ecr.aws/x6n7b2o2/newrelic-lambda-layers-python)
+ * [Extension-only](https://gallery.ecr.aws/newrelic-lambda-layers-for-docker/newrelic-lambda-layers-lambdaextension)
+3. **At the top of your Dockerfile** add the following code: 
+
+ <Tabs>
+        <TabsBar>
+                <TabsBarItem id="1">Dockerfile for Java</TabsBarItem>
+                <TabsBarItem id="2">Dockerfile for Node.js</TabsBarItem>
+                <TabsBarItem id="3">Dockerfile for Python</TabsBarItem>
+                <TabsBarItem id="4">Dockerfile for Extension-only</TabsBarItem>
+        </TabsBar>
+
+        <TabsPages>
+        <TabsPageItem id="1">
+                ```
+                FROM public.ecr.aws/newrelic-lambda-layers-for-docker/newrelic-lambda-layers-java:17-arm64 AS layer
+
+                # Layer code
+                COPY --from=layer /opt/ /opt/
+                ```
+        </TabsPageItem>
+
+        <TabsPageItem id="2">
+                ```
+                FROM public.ecr.aws/newrelic-lambda-layers-for-docker/newrelic-lambda-layers-python:38-arm64 AS layer
+
+                # Layer code
+                COPY --from=layer /opt/ /opt/
+                ```
+        </TabsPageItem>
+
+        <TabsPageItem id="3">
+                ```
+                FROM public.ecr.aws/newrelic-lambda-layers-for-docker/newrelic-lambda-layers-nodejs:20-arm64 AS layer
+
+                # Layer code
+                COPY --from=layer /opt/ /opt/
+                ```
+        </TabsPageItem>
+
+        <TabsPageItem id="4">
+                ```
+                FROM public.ecr.aws/newrelic-lambda-layers-for-docker/newrelic-lambda-layers-lambdaextension:x86 as layer
+
+                # Layer code
+                COPY --from=layer /opt/ /opt/
+                ```
+        </TabsPageItem>
+        </TabsPages>
+</Tabs>
+
+</Step>
+<Step>
+
+### Create an image
+        1. After you add the New Relic pre-built layer to your Dockerfile, you'll need to build a new image from the Dockerfile. This new image will have your original code _and_ New Relic's layer. To do this, we recommend following the steps listed in the AWS guide, [Pushing a Docker image](https://docs.aws.amazon.com/AmazonECR/latest/userguide/docker-push-ecr-image.html).
```

Comment:
- i might consider making this step more expository, rather than bulleted out. I'm not sure if customers need to be told to push the image if that step is already implied in the AWS steps
- In the AWS docs, it says that customers need to push the image to the repository they've created. Does "AWS Elastic Container Registry (ECR)" refer to what they created by running those commands in step 1?